src/main/java/org/jboss/windup/plugin/WindupMojo.java

```diff
@@ -73,7 +79,7 @@
     /**
      * Location of the input file application.
      */
-    @Parameter(alias = "input", property = "input", required = true)
+    @Parameter( defaultValue = "${project.build.sourceDirectory}", property = "input", required = true )
```

Comment:
from Maven documentation for `alias`:

> Specifies an alias which can be used to configure this parameter from the POM. This is primarily useful to improve user-friendliness, where Mojo field names are not intuitive to the user or are otherwise not conducive to configuration via the POM.

So It is for pom.xml plugin configuration, while property is for using on CLI like `-D<property>=something` and name should automatically be detected from field name.